package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "node_modules/.bin/eslint ./ || true",
     "test": "nyc --reporter=html --reporter=text mocha --require babel-core/register ./src/tests/ || true",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test"
+    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test",
+    "start": "nodemon ./bin/www/ --exec babel-node "
```

Comment:
Did you run `npm start`? Did the app start? Were yu able to make `CRUD` requests from `postman`?

I think The final `/` after `www` should not be there. Also, i'm not sure there should be the space that exists before the closing quotation mark just after `babel-node`

Reply:
okay. why am I not surprised you are awake. lol. Thank you. Yes it all worked